config/config.go

```diff
@@ -208,6 +209,7 @@ type Config struct {
 	EnableSwarm bool `yaml:"enable-swarm"`
 	// redis based
 	SwarmRedisURLs         *listFlag     `yaml:"swarm-redis-urls"`
```

Comment:
SwarmRedisURLs usage string should mention env variable and config option like here https://github.com/zalando/skipper/commit/fbc0d7a583a080d6df3b5eb5de57b2a35ee23ca7#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R449

Reply:
Cool. Thanks